spec.bs

```diff
@@ -533,6 +577,9 @@ The [=steps to respond to a payment request=] for this payment method, for a giv
     "{{AuthenticationExtensionsPaymentInputs/instrument}}" → |data|.{{SecurePaymentConfirmationRequest/instrument}}]».
 
     <div class="note">**TODO**: We do not have the RP origin at this step; maybe that should just go in the extension processing steps?</div>
+    Note: This is set in WebAuthn Authenticator, as part of creation? How do we validate/ensure we're not jumping to an incorrect domain? is this currently validated by the browser agent or by the CTAP protocol? Then we need to set it in some way?
+
+    Q: is payeeOrigin optional? What if it's not set? Do we allow it to be empty?
```

Comment:
It is not optional; https://w3c.github.io/secure-payment-confirmation/#dictdef-securepaymentconfirmationrequest